### PR TITLE
Update webgpu-inter-stage-variables.html

### PR DESCRIPTION
**Pull request by shallow-beach**
• c/e
• clarify/restate important points
• punctuation before code when appropriate

### CODE REVIEW

**Comment by shallow-beach** [comment]
oops recommited to fix small error on last 2 lists (make use of punctuation consistent)

**Comment by shallow-beach** [comment]
another [recommit](./144/commits/a1b3c9ce3e9a61f7bdfd396d84d5ad541dc3dbb4) to scrub incorrect clarification, just c/e

File: webgpu/lessons/webgpu-inter-stage-variables.md [hunk 1]
```diff
@@ -308,18 +308,20 @@ And we'd have to update our pipeline creation to use these
...
-things like structures or binding and group locations twice if two or more shaders
-share bindings or structures or constants or functions. But, from the POV of WebGPU,
-it's as though you did duplicate all of them, once for each entryPoint.
+sure it's syntactically correct. Then, WebGPU looks at each `entryPoint`
+you specify, separately. It looks at the parts that each entryPoint references
+and nothing else. 
+
+Shared strings are useful because you don't have to type things like structures, 
+binding and group locations, constants, and functions twice if multiple shaders 
+share the resources/entities. But, from the POV of WebGPU, it's as though you did duplicate
```
Comment (greggman) [inline webgpu/lessons/webgpu-inter-stage-variables.md:323]: > if multiple shaders share the resources/entities

I don't know what resources and entities are here. 

How about

> Shared strings are useful because multiple shaders can then share things like structures, binding and group locations, constants, and functions.

**Review by greggman — CHANGES_REQUESTED** [review]
Thanks for these. A couple of comments.

File: webgpu/lessons/webgpu-inter-stage-variables.md [hunk 2]
```diff
@@ -308,18 +308,20 @@ And we'd have to update our pipeline creation to use these
 
 ```
 
-And this would also work
+These modifications render the same:
```
[the remark this answers is not in the document]
Comment (shallow-beach) [reply webgpu/lessons/webgpu-inter-stage-variables.md:311]: I just used 'render' as a similar verb for 'work'; your reasoning definitely makes sense to me though.

Maybe suggest something like:

>These changes work the same:

Also could just skip any word change (I tend to be too wordy anyway) and just append the colon:

>- And this would also work
>+ And this would also work:

File: webgpu/lessons/webgpu-inter-stage-variables.md [hunk 3]
```diff
@@ -154,18 +154,18 @@ That field is **NOT** an inter-stage variable. Instead, it's a `builtin`. It
...
+In a vertex shader `@builtin(position)` is the output of a coordinate that the GPU uses to draw
+triangles/lines/points.
 
 In a fragment shader, `@builtin(position)` is an input. It's the pixel coordinate
 of the pixel that the fragment shader is currently being asked to compute a color
 for.
 
-Pixel coordinates are specified by the edges of pixels. The values provided to
-the fragment shader are the coordinates of the center of the pixel
+The values provided *by* the vertex shader are the coordinates of the edges of pixels. The values provided *to*
```
[the remark this answers is not in the document]
Comment (shallow-beach) [reply webgpu/lessons/webgpu-inter-stage-variables.md]: thank you for clarifying :)

that was definitely not a valid edit

File: webgpu/lessons/webgpu-inter-stage-variables.md (hunk 1 above)
Comment (shallow-beach) [reply webgpu/lessons/webgpu-inter-stage-variables.md:323]: much cleaner, i agree. 'resources/entities' being used nebulously + placeholder-y.

**Comment by shallow-beach** [comment]
apologies i'm new to GitHub: how to handle 'resolve conversations'? should i make fixing edits then 'resolve' for re-review? (i resolved one comment on an old commit)

**Comment by greggman** [comment]
I'm not sure there's a standard way.

Generally you'd push some changes to this branch. If you think you've addressed the issue with your changes then click "resolve conversation".

File: webgpu/lessons/webgpu-inter-stage-variables.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shallow-beach) [reply webgpu/lessons/webgpu-inter-stage-variables.md]: [resolution commit](../pull/144/commits/45361680a6419ad33822d63aec48e56d6b53491b)

File: webgpu/lessons/webgpu-inter-stage-variables.md (hunk 1 above)
Comment (shallow-beach) [reply webgpu/lessons/webgpu-inter-stage-variables.md]: [resolution commit](./144/commits/b7c35efa2a8bd63ce633ae1b5ad9b0ab0128e57e)

File: webgpu/lessons/webgpu-inter-stage-variables.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (shallow-beach) [reply webgpu/lessons/webgpu-inter-stage-variables.md]: [resolution commit](./144/commits/a1b3c9ce3e9a61f7bdfd396d84d5ad541dc3dbb4)

**Comment by shallow-beach** [comment]
> I'm not sure there's a standard way.
> 
> Generally you'd push some changes to this branch. If you think you've addressed the issue with your changes then click "resolve conversation".

Resolved with commit links in each thread 🫡
